session-fedora-pod is not having "labels" key when created

session-fedora-pod is not having "labels" key when created.
Due to this, it is getting failed for below code
```
labels = dpod.get().get("metadata").get("labels")
if not any("deployer-pod-for" in label for label in labels):
   if pod_name in dpod.name:
         return dpod
```
Error:
```
E               TypeError: 'NoneType' object is not iterable
```
RP link: https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/all/27446/1355792/1355866/log?item1Params=page.sort%3Dstatus%252CASC

--- a/ocs_ci/templates/app-pods/fedora_with_linuxtar_files.yaml
+++ b/ocs_ci/templates/app-pods/fedora_with_linuxtar_files.yaml
@@ -3,6 +3,8 @@ kind: Pod
 metadata:
   name: fedora
   namespace: openshift-storage
+  labels:
+    app: fedora-pod
 spec:
   containers:
     - name: fedora

